static/js/containers/PostPage.js

```diff
@@ -178,7 +181,14 @@ class PostPage extends React.Component<*, void> {
       ])
 
       if (post.url) {
-        await dispatch(actions.embedly.get(post.url))
+        const embedlyResponse = await dispatch(actions.embedly.get(post.url))
+
+        if (
```

Comment:
This seems to be identical to code in `CreatePostPage`,  maybe it could be moved to a common function in `lib/embed.js`?

```javascript
export const loadTwitter = (embedlyResponse) => {
  if (embedlyResponse && embedlyResponse.response.provider_name === "Twitter") {
    window.twttr.widgets.load()
  }
}
```

Reply:
yeah, that's a good idea